src/samplers/cpu/stats.rs

```diff
@@ -41,3 +41,19 @@ gauge!(
 
 counter!(CPU_CYCLES, "cpu/cycles");
 counter!(CPU_INSTRUCTIONS, "cpu/instructions");
+gauge!(IPKC, "cpu/ipkc", "Average Instructions Per Thousand Cycle");
+gauge!(
+    IPKNS,
```

Comment:
Is this a conventional way of expressing this? Why is it not IPUS (Instructions per Microsecond)? Is the scaling here for the same reason, to avoid loss of precision when representing as a i64 instead of a float? What's the benefit of exporting this separately from `cpu/instructions` and doing a `rate()` in the obs stack?